arex-instrumentation/dynamic/arex-dynamic-common/src/main/java/io/arex/inst/dynamic/common/DynamicClassExtractor.java

```diff
@@ -308,6 +316,12 @@ Object restoreResponse(Object result) {
             return Mono.justOrEmpty(result);
         }
 
+        if (FLUX.equals(this.methodReturnType)) {
+            if (result instanceof Throwable) {
+                return Flux.error((Throwable) result);
+            }
+            return result == null ? Flux.empty() : FluxConsumer.restore((result));
```

Comment:
Just `return FluxConsumer.restore((result));`, the rest of the logic can be wrapped in `restore` method